### DISCUSSION TITLE
Doubt about function Rainforest Project

### DISCUSSION THREAD

**Discussion by anicolco** [mainIine/foe-helfer-extension#2646] [Q&A] [answered]
Hello.
Something I miss is knowing if the "Rain Forest Project" building is functioning well. I had removed it and put it back for the plans of the large buildings on Titan, as I have the impression that they will be in high demand.

Do you know if there is any way to find out if it's working or if there is any plugin? 

Thank you very much.

**Comment by Beelzebob6666** [comment, marked as answer]
the only way to know for sure would be to track BP gotten from aiding over a long period - not really worth the effort in my opinion

**Reply by anicolco** [reply]
Thank you. Are you referring to counting them manually?

**Reply by Beelzebob6666** [reply]
no matter how you track it ;)
counting it with the helper might have other problems - like not knowing how many players were actually aided (not sure if the reward report has an info on that)
